### DISCUSSION TITLE
Modify Output Directory

### DISCUSSION THREAD

**Discussion by jtprichard** [Nice3point/RevitTemplates#60] [Q&A] [answered]
First, thank you for publishing and making this available.
Is there a way to modify the output directory for both the debug compiler and the WIX build?  I am trying to insert a common directory under the Revit version so that I can consolidate multiple solution dlls in a single folder, and eliminate the project folder.

ie:  .../Revit/Addins/2022/PerformanceBIM/[application DLLs]
where my solution structure is
[Solution1]/[Addin1]
[Solution2]/[Addin2]
[Solution3]/[Common Support Project Referenced by Solutions 1&2]

I'm fine manually changing the manifest file.  Just trying to figure out how and where the [Addin] folder is inserted into the output directory structure.

**Comment by Nice3point** [comment, marked as answer]
This is the responsibility of the Revit.Build.Tasks package
You can disable the default publishing, and configure it in msbuid yourself

https://github.com/Nice3point/RevitTemplates/wiki/Third-party-files
https://github.com/Nice3point/Revit.Build.Tasks

**Reply by jtprichard** [reply]
Thank you. I knew I had to be missing something easy.
